test/functional/wallet_keypool_topup.py

```diff
@@ -40,32 +40,37 @@ def run_test(self):
         shutil.copyfile(wallet_path, wallet_backup_path)
         self.start_node(1, self.extra_args[1])
         connect_nodes_bi(self.nodes, 0, 1)
+        connect_nodes_bi(self.nodes, 0, 2)
+        connect_nodes_bi(self.nodes, 0, 3)
 
-        self.log.info("Generate keys for wallet")
-        for _ in range(90):
-            addr_oldpool = self.nodes[1].getnewaddress()
-        for _ in range(20):
-            addr_extpool = self.nodes[1].getnewaddress()
+        for i, output_type in enumerate(["legacy" , "p2sh-segwit", "bech32"]):
 
-        self.log.info("Send funds to wallet")
-        self.nodes[0].sendtoaddress(addr_oldpool, 10)
-        self.nodes[0].generate(1)
-        self.nodes[0].sendtoaddress(addr_extpool, 5)
-        self.nodes[0].generate(1)
-        sync_blocks(self.nodes)
+            self.log.info("Generate keys for wallet with address type: " + output_type)
```

Comment:
Make this logging lazy by using string format arguments :-)

Reply:
done